recipes/archicad-apidevkit/all/conanfile.py

```diff
@@ -0,0 +1,52 @@
+from conans import tools
+from conan import ConanFile
+from conan.tools.files import copy, get
+from conan.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.46.0"
+
+
+class ArchicadApidevkitConan (ConanFile):
+    name = "archicad-apidevkit"
+    description = "The General API Development Kit enables software developers to extend the functionality of Archicad"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://archicadapi.graphisoft.com/"
+    license = "LicenseRef-LICENSE"
+    settings = "os", "arch", "build_type", "compiler"
+    no_copy_source = True
+    topics = "api", "archicad", "development"
+    short_paths = True
+
+    @property
+    def _acdevkit_arch(self):
+        return str(self.settings.arch)
+
+    def package_id(self):
+        del self.info.settings.compiler
+        del self.info.settings.build_type
+
+    def package_info(self):
+        self.cpp_info.bindirs = []
+        self.cpp_info.frameworkdirs = []
+        self.cpp_info.libdirs = []
+        self.cpp_info.includedirs = []
```

Comment:
`self.env_info.AC_API_DEVKIT_DIR` should be set to the appropriate directory, since that's the documented usage anyway. This will make the variable available to the virtual environment generators.

Reply:
It should also go in build info, env info is deprecated 

Ideal this should be added as a tool requires and the called as intented to build something and run the output